src/Apps/Artist/Routes/Overview/Components/RecommendedArtist.tsx

```diff
@@ -0,0 +1,188 @@
+})
+class RecommendedArtistWithTracking extends React.Component<
+  RecommendedArtistProps
+> {
+  @track({
+    type: Schema.Type.Thumbnail,
+    action_type: Schema.ActionType.Click,
+  })
+  trackArtworkClicked() {
+    // noop
+  }
+
+  render() {
+    return (
+      <RecommendedArtist
+        {...this.props}
+        onArtworkClicked={this.trackArtworkClicked.bind(this)}
+      />
+    )
+  }
+}
+
+const RecommendedArtist: FC<
+  RecommendedArtistProps & { onArtworkClicked: () => void }
+> = ({ artist, onArtworkClicked }) => {
+  const { user, mediator } = useContext(SystemContext)
+
+  return (
+    <>
+      <EntityHeader
+        mt={4}
+        imageUrl={get(artist, a => a.image.cropped.url, "")}
+        name={artist.name}
+        meta={artist.formatted_nationality_and_birthday}
+        href={artist.href}
+        FollowButton={
+          <FollowArtistButton
+            artist={artist}
+            user={user}
+            trackingData={{
+              modelName: Schema.OwnerType.Artist,
+              context_module: Schema.ContextModule.RecommendedArtists,
+              entity_id: artist._id,
+              entity_slug: artist.id,
+            }}
+            onOpenAuthModal={() => handleOpenAuth(mediator, artist)}
+            render={({ is_followed }) => {
+              return (
+                <Sans
+                  size="2"
+                  weight="medium"
+                  color="black"
+                  style={{
+                    cursor: "pointer",
+                    textDecoration: "underline",
+                  }}
+                >
+                  {is_followed ? "Following" : "Follow"}
+                </Sans>
+              )
+            }}
+          />
+        }
+      />
+
+      <Spacer mb={3} />
+
+      <Carousel
+        data={artist.artworks_connection.edges as object[]}
+        render={artwork => {
+          const aspect_ratio = get(artwork, a => a.node.image.aspect_ratio, 1)
+
+          return (
+            <FillwidthItem
+              artwork={artwork.node}
+              targetHeight={HEIGHT}
+              imageHeight={HEIGHT}
+              width={HEIGHT * aspect_ratio}
+              margin={10}
+              user={user}
+              mediator={mediator}
+              onClick={onArtworkClicked}
+            />
+          )
+        }}
+      />
+    </>
+  )
+}
+
+export const RecommendedArtistFragmentContainer = createFragmentContainer(
+  RecommendedArtistWithTracking,
+  graphql`
+    fragment RecommendedArtist_artist on Artist {
+      id
+      _id
+      name
+      formatted_nationality_and_birthday
+      href
+      image {
+        cropped(width: 100, height: 100) {
+          url
+        }
+      }
+      artworks_connection(first: 20) {
+        edges {
+          node {
+            __id
+            image {
+              aspect_ratio
+            }
+            ...FillwidthItem_artwork @relay(mask: false)
```

Comment:
Was this needed? At first glance this _seems_ correct but everything should work w/o the mask directive.